styles.css

```diff
@@ -1 +1,56 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+body {
+    background-color: hsl(56, 97%, 87%);
+}
+
+.container {
+    display: grid;
+    grid-template-rows: 50px 190px 100px auto 50px;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 16px;
+    grid-template-areas:
+            "a c c"
+            "b c c"
+            "b e d"
+            "b e x"
+            "f f f";
+}
+
+.item {
```

Comment:
je kan in .item voor alle grid-cellen nog padding toevoegen (bijvoorbeeld 15px). In opdracht zie je in de cel B, waar text in zit, de ruimte tussen text en de border.